components/command/command-handler/src/main/java/uk/gov/justice/services/core/aggregate/AggregateService.java

```diff
@@ -26,38 +32,50 @@
     Logger logger;
 
     @Inject
+    SnapshotService snapshotService;
```

Comment:
As this change will involve changing the producer factory for Aggregate Service, Discussing with Shaun plan is to revisit the branch for this change once these review changes have been accepted